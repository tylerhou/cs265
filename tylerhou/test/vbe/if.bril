@main {
    cond: bool = const true;
    a: int = const 1;
    b: int = const 2;
    br cond .left .right;
.left:
    c: int = add a b;
    jmp .end;
.right:
    c: int = add a b;
    jmp .end;
.end:
    print c;
}
