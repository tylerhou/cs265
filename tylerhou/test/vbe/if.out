@main {
._special_entry_for_ssa:
  376807418.0: int = const 1;
  376807418.0: int = const 2;
  376807418.0: bool = const true;
.__b0:
  cond.0: bool = const true;
  a.0: int = const 1;
  b.0: int = const 2;
  376807418.1: int = add a.0 b.0;
  br cond.0 .left .right;
.left:
  c.0: int = add a.0 b.0;
  jmp .end;
.right:
  c.1: int = add a.0 b.0;
  jmp .end;
.end:
  c.2: int = phi c.0 c.1 .left .right;
  print c.2;
}
